torch/csrc/distributed/c10d/ProcessGroupNCCL.hpp

```diff
@@ -288,8 +286,7 @@ class TORCH_API ProcessGroupNCCL : public Backend {
 
     // Wrapper method for the static checkForNCCLErrors which can be overridden
     // for tests.
-    virtual std::exception_ptr checkForNCCLErrors(
-        const std::vector<std::shared_ptr<NCCLComm>>& ncclComms) const;
+    virtual std::exception_ptr checkForNCCLErrors();
```

Comment:
Because the compilation would otherwise fail. 
Note that we removed the `ncclComms` argument here, and will be checking `self.ncclComm_` (which is the same thing). When we touch `self.ncclComm_`, it seems the compiler complains if const is there.

Reply:
Yes, since the member variable is passed to a static func, that's fine